File: AMS_基础信息组件/code/client/source/YssInformation.Support/Sys/ConvertDict/Zdorg/Pojo/ZdOrgTreeView.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

using Newtonsoft.Json;

namespace YssInformation.Support.Sys.ConvertDict.Zdorg.Pojo
{
    public class ZdOrgTreeView : ZdCorpOrg
    {
        /// <summary>
        /// 树型层级
        /// </summary>
        private int n_Level = 1;

        /// <summary>
        /// 节点代码
        /// </summary>
        private string nodeCode = "";

        /// <summary>
        /// 父节点代码
        /// </summary>
        private string parentCode = "";

        /// <summary>
        /// 树型层级
        /// </summary>
        [JsonProperty(PropertyName = "n_Level")]
        public int N_Level
        {
            get { return n_Level; }
            set { n_Level = value; }
        }

        /// <summary>
        /// 节点代码
        /// </summary>
        [JsonProperty(PropertyName = "nodeCode")]
        public string NodeCode
        {
            get { return nodeCode; }
            set { nodeCode = value; }
        }

        /// <summary>
        /// 父节点代码
        /// </summary>
        [JsonProperty(PropertyName = "parentCode")]
        public string ParentCode
        {
            get { return parentCode; }
            set { parentCode = value; }
        }
    }
}

File: AMS_基础信息组件/code/server/source/com.yss.ams.base.information.support/src/main/java/com/yss/ams/base/information/support/sys/convertdict/zdorg/pojo/ZdOrgTreeView.java
package com.yss.ams.base.information.support.sys.convertdict.zdorg.pojo;


public class ZdOrgTreeView extends ZdCorpOrg{

	private static final long serialVersionUID = 1L;
	
	private int n_Level = 1;
	
	private String nodeCode;
	
	private String parentCode;

	public int getN_Level() {
		return n_Level;
	}

	public void setN_Level(int nLevel) {
		n_Level = nLevel;
	}

	public String getNodeCode() {
		return nodeCode;
	}

	public void setNodeCode(String nodeCode) {
		this.nodeCode = nodeCode;
	}

	public String getParentCode() {
		return parentCode;
	}

	public void setParentCode(String parentCode) {
		this.parentCode = parentCode;
	}

	public static long getSerialversionuid() {
		return serialVersionUID;
	}

}
